universe/storage/s3/internal/prepare/prepares3.go

```diff
@@ -227,11 +221,7 @@ func (provisionHook) Apply(ctx context.Context, req provisioning.StackRequest, o
 			&schema.BinaryConfig_EnvEntry{Name: "MINIO_PASSWORD", FromSecretRef: schema.MakePackageRef("namespacelabs.dev/foundation/universe/storage/minio/creds", "root-password")})
 		commonArgs = append(commonArgs, fmt.Sprintf("--%s=%s", useMinioFlag, service))
 	} else {
-		for _, secret := range col.SecretsOf("namespacelabs.dev/foundation/universe/aws/client") {
-			if secret.Name == "aws_credentials_file" {
-				initArgs = append(initArgs, fmt.Sprintf("--aws_credentials_file=%s", secret.FromPath))
-			}
-		}
+		return fmt.Errorf("AWS S3 is not supported")
```

Comment:
I think you should just remove this path; this is only trying to forward --aws_credentials_file if its set. No forward is done anymore, but AWS S3 is still supported, but requires irsa.

Reply:
Done.